### PR TITLE
Surf dcm presets

### PR DESCRIPTION
**Pull request by internetova**

### CODE REVIEW

**Review by Sadhorsephile — APPROVED** [review]

File: lib/recommended.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dart_code_metrics:
+  rules:
+    # Style
+    - unnecessary-trailing-comma: false
```
Comment (maxim-sysoev) [inline lib/recommended.yaml]: this file looks too empty

File: lib/all.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+dart_code_metrics:
+  extends:
+    - package:surf_dcm_presets/dart.yaml
+    - package:surf_dcm_presets/flutter.yaml
```
Comment (maxim-sysoev) [inline lib/all.yaml:4]: there most likely should be rules for an intl, provider and others

Comment (internetova) [reply lib/all.yaml:4]: ✅

File: lib/recommended.yaml (hunk 1 above)
Comment (internetova) [reply lib/recommended.yaml]: ✅

File: lib/flutter.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    - prefer-single-widget-per-file:
+        ignore-private-widgets: true
+    - prefer-sliver-prefix
+    - prefer-text-rich
+    - prefer-using-list-view
+    - prefer-widget-private-members
+    - proper-super-calls
+    - use-setstate-synchronously
+    # Style
+    - unnecessary-trailing-comma: false
```
Comment (maxim-sysoev) [inline lib/flutter.yaml]: its more about dart

**Review by maxim-sysoev — APPROVED** [review]

**Review by maxim-sysoev — APPROVED** [review]

**Review by mpkander — APPROVED** [review]

**Pull request merged**